app/community/page.tsx

```
@@ -0,0 +1,374 @@
+
+    const allowance = useReadContract({
+        abi: commonsConfig.abi,
+        address: TOKEN_ADDRESS as `0x${string}`,
+        functionName: "allowance",
+        args: [account.address as `0x${string}`, nftConfig.address as `0x${string}`],
+        query: {
+            refetchInterval: 1,
+        },
+    });
+
+    const allowanceAvailable = allowance && allowance.data ? BigInt(allowance.data as bigint) : BigInt(0);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)